Dockerfile

```
@@ -0,0 +1,24 @@
+RUN apt-get update && apt-get install -y \
+    wget \
+    curl \
+    jq \
+    sqlite3 \
+    && rm -rf /var/lib/apt/lists/*
+
+COPY requirements.txt .
+RUN pip install --no-cache-dir -r requirements.txt
+
+# Install lldap-cli
+RUN wget https://raw.githubusercontent.com/Zepmann/lldap-cli/refs/heads/main/lldap-cli -O /usr/local/bin/lldap-cli && \
```

Hadolint (2.12.0)

[info] Avoid use of wget without progress bar. Use `wget --progress=dot:giga <url>`. Or consider using `-q` or `-nv` (shorthands for `--quiet` or `--no-verbose`).

(DL3047)